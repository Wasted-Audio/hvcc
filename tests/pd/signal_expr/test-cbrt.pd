#N canvas 827 239 734 565 12;
#X obj 60 47 loadbang;
#X obj 60 74 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 60 345 print;
#X msg 76 161 0.5;
#X msg 109 161 3.7;
#X msg 143 161 -5;
#X obj 143 222 sig~;
#X obj 60 313 snapshot~;
#X obj 60 108 t_delays;
#X obj 143 256 expr~ cbrt($v1);
#X connect 0 0 1 0;
#X connect 1 0 8 0;
#X connect 3 0 6 0;
#X connect 4 0 6 0;
#X connect 5 0 6 0;
#X connect 6 0 9 0;
#X connect 7 0 2 0;
#X connect 8 0 7 0;
#X connect 8 1 3 0;
#X connect 8 2 7 0;
#X connect 8 3 4 0;
#X connect 8 4 7 0;
#X connect 8 5 5 0;
#X connect 9 0 7 0;
