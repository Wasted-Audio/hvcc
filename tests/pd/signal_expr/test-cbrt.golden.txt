[@ 2.021] print: -1.70998
[@ 6.021] print: 1.54668
[@ 10.021] print: 0.793701
